content/SCALE/SCALETutorials/Dashboard/SCALETimeSync.md

```diff
@@ -7,13 +7,30 @@ tags:
 - scaletimesync
 ---
 
-TrueNAS SCALE provides the ability to synchronize SCALE and system server time when they get out of sync with each other. 
-This function is not designed to correct time differences weeks out of alignment.
+TrueNAS SCALE allows users to synchronize SCALE and system server time when they get out of sync. 
+This function will not correct time differences over 30 days out of alignment.
 
-The **System Information** widget on the main **[Dashboard]({{< relref "/SCALE/SCALEUIReference/SCALEDashboard.md" >}})** displays a message and provides an icon button that executes the time-synchronization operation only when SCALE detects a discrepancy between SCALE and system server time.
+The **System Information** widget on the **[Dashboard]({{< relref "/SCALE/SCALEUIReference/SCALEDashboard.md" >}})** displays a message and provides an icon button that executes the time-synchronization operation only when SCALE detects a discrepancy between SCALE and system server time.
 
 ![DashboardSytemInfoWidgetTimeSync](/images/SCALE/22.12/DashboardSytemInfoWidgetTimeSync.png "System Information Widget with Time Sync")
 
 Click the **Synchronize Time** <span class="material-icons">loop</span> icon button to initiate the time-synchronization operation.
 
-{{< taglist tag="scaletimesync" limit="10" >}}
\ No newline at end of file
+{{< expand "What if my time is off by more than 30 days?" "v" >}}
+If your time is off by more than 30 days, TrueNAS SCALE does not allow you to sync since the system probably has one of the following underlying issues:
+
+* The BIOS timezone is incorrect
+* The motherboard CMOS battery is failing
+
+To check the BIOS timezone, reboot your system. During boot, press the indicated key that takes you to the BIOS setup screen. The key differes based on manufacturer (<kbd>F2</kbd>, <kbd>Delete</kbd>, <kbd>Esc</kbd>, etc.). If you don't know which key to use, check the manufacturer documentation for your server.
+
+Once you have entered the BIOS setup, ensure the timezone is UTC. If not, set it to UTC, save the configuration changes, and reboot the system.
```

Comment:
After instead of once is a better fit because they do this after entering the BIOS setup
```suggestion
After you enter the BIOS setup, ensure the timezone is UTC. If not, set it to UTC, save the configuration changes, and reboot the system.
```